Add Kopete

Mailaender opened the pull request:
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://matrix.to/#/#flatpak:kde.org
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)* https://invent.kde.org/network/kopete/-/merge_requests/5

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

Mailaender commented:
bot, build org.kde.kopete

hfiguiere commented on org.kde.kopete.json:
```diff
@@ -0,0 +1,270 @@
+    "--device=dri",
+    "--share=ipc",
+    "--share=network",
+    "--socket=x11",
+    "--socket=wayland",
```
```suggestion
    "--socket=fallback-x11",
    "--socket=wayland",
```

hfiguiere commented:
cc @flathub/kde

barthalion commented:
bot, build org.kde.kopete

barthalion commented:
Waiting for approval from @flathub/kde.

travier commented:
Manifest LGTM. Not yet tested.

tsdgeos commented:
There's lots of not found dependencies, can you try adding them? or saying why we don't need them?

-- Could not find Kleopatra includes.
-- Could not find Kleopatra library.

-- Checking for module 'meanwhile'
--   Package 'meanwhile', required by 'virtual:world', not found

-- Checking for module 'ortp'
--   Package 'ortp', required by 'virtual:world', not found

-- Checking for module 'mediastreamer>=2.3.0'
--   Package 'mediastreamer', required by 'virtual:world', not found

-- Not found SRTP

Mailaender commented:
I don't understand why it does not find libkleo as I added that. Meanwhile is unmaintained and requires multiple patches to build. It can be added, but I tried to get it to build at all as a first priority. ortp and mediastreamer have complex dependencies so I postponed it. I don't understand why srtp is not found as I added it. Probably because ortp and friends are missing.

travier commented:
Overall, I'm afraid that this is an unmaintained app. Maybe only worth beta.

tsdgeos commented:
> Overall, I'm afraid that this is an unmaintained app. Maybe only worth beta.

Please read https://tsdgeos.blogspot.com/2021/06/everything-we-release-in-kde-gear-is.html

Mailaender commented:
I think beta is a good choice because the protocols like ICQ and AIM don't work anymore.

travier commented:
> Please read [tsdgeos.blogspot.com/2021/06/everything-we-release-in-kde-gear-is.html](https://tsdgeos.blogspot.com/2021/06/everything-we-release-in-kde-gear-is.html)

Indeed, sorry.

tsdgeos commented:
Though i will not disagree to put it in Beta if both of you think it's what makes sense :)

nedrichards commented:
/merge:beta
